### PR TITLE
storaged: Move 'create' buttons into overview panels.

### PR DESCRIPTION
**Pull request by mvollmer**
And always show the panels even when they are empty.

### CODE REVIEW

**Comment by mvollmer** [comment]
@andreasn, please check this out.

**Comment by stefwalter** [comment]
When the panels are empty they should show a helpful message, like a ```well``` in the empty state similar to what we do on the kubernetes dashboard. @andreasn, WDYT?

**Comment by andreasn** [comment]
storage-panels

**Comment by andreasn** [comment]
That would take up a lot of space, pushing down the disks a bit too much in my taste.

**Comment by andreasn** [comment]
Two variants of wells.
One compact and one regular.

mockup

**Comment by andreasn** [comment]
And having the non-compact well just taking up one extra row might be OK.

**Comment by dperpeet** [comment]
> And having the non-compact well just taking up one extra row might be OK.

I like v1 (more compact), but maybe with a bit of reduced redundancy and hinting at possible actions.

i.e.
Volume Groups [rest in grey] None, add one
iSCSI Targets [rest in grey] None, add one
RAID Devices [rest in grey] None, add one

While discussing this on irc with @fabiand we didn't come up with a good, compact way of including a reference to the iSCSI rename button in the text. I think it's enough to mention to primary action in the text.

**Comment by mvollmer** [comment]
Looks like this now:

storage-panels

**Comment by dperpeet** [comment]
> Looks like this now:



> Looks like this now:

Looks good - especially since the background of the extra info is white. This is compact enough for me and I like that the title bar doesn't change.

**Comment by mvollmer** [comment]
Ready.

File: pkg/storaged/index.html [hunk 1]
```diff
@@ -66,9 +66,14 @@ <h1 translatable="yes">The "storaged" API is not available on this system.</h1>
   <!-- OVERVIEW -->
 
   <script id="mdraids-tmpl" type="x-template/mustache">
-    {{#HasMDRaids}}
     <div class="panel panel-default">
-      <div class="panel-heading" translatable="yes">RAID Devices</div>
+      <div class="panel-heading">
+        <span style="float:right">
```
Comment (stefwalter) [inline pkg/storaged/index.html]: The patternfly way is to use ```class='pull-right'``` instead of inline styles.

File: pkg/storaged/index.html [hunk 2]
```diff
@@ -87,14 +92,22 @@ <h1 translatable="yes">The "storaged" API is not available on this system.</h1>
...
     </div>
-    {{/HasMDRaids}}
   </script>
 
   <script id="vgroups-tmpl" type="x-template/mustache">
-    {{#HasVGroups}}
     <div class="panel panel-default">
-      <div class="panel-heading" translatable="yes">Volume Groups</div>
+      <div class="panel-heading">
+        <span style="float:right">
```
Comment (stefwalter) [inline pkg/storaged/index.html]: pull-right

File: pkg/storaged/index.html [hunk 3]
```diff
@@ -66,9 +66,14 @@ <h1 translatable="yes">The "storaged" API is not available on this system.</h1>
   <!-- OVERVIEW -->
 
   <script id="mdraids-tmpl" type="x-template/mustache">
-    {{#HasMDRaids}}
     <div class="panel panel-default">
-      <div class="panel-heading" translatable="yes">RAID Devices</div>
+      <div class="panel-heading">
+        <span style="float:right">
+          <button id="create-mdraid" class="btn btn-primary">+</button>
```
Comment (stefwalter) [inline pkg/storaged/index.html]: This should have class='fa fa-plus', and no content.

File: pkg/storaged/index.html [hunk 4]
```diff
@@ -87,14 +92,22 @@ <h1 translatable="yes">The "storaged" API is not available on this system.</h1>
...
-    {{/HasMDRaids}}
   </script>
 
   <script id="vgroups-tmpl" type="x-template/mustache">
-    {{#HasVGroups}}
     <div class="panel panel-default">
-      <div class="panel-heading" translatable="yes">Volume Groups</div>
+      <div class="panel-heading">
+        <span style="float:right">
+          <button id="create-volume-group" class="btn btn-primary">+</button>
```
Comment (stefwalter) [inline pkg/storaged/index.html]: This should have class='fa fa-plus', and no content.

**Comment by mvollmer** [comment]
Fixed the button markup issues.

**Pull request closed without merging**